.github/CODEOWNERS

```
@@ -2,21 +2,132 @@
+/enterprise/cmd/frontend @beyang
+/enterprise/cmd/server @beyang
+/enterprise/dev @beyang
+/cmd/frontend/docsite @sqs
+/cmd/frontend/shared @beyang
+/cmd/frontend/backend @beyang
+/cmd/frontend/internal/app/assets @beyang
+/cmd/frontend/internal/app/templates @beyang
+/cmd/frontend/internal/app/canonicalurl @beyang
+/cmd/frontend/internal/app/*.go @beyang
+/cmd/frontend/internal/app/assetsutil @beyang
+/cmd/frontend/internal/app/ui @beyang
+/cmd/frontend/internal/app/returnto @beyang
+/cmd/frontend/internal/app/pkg @beyang
+/cmd/frontend/internal/app/router @beyang
+/cmd/frontend/internal/app/errorutil @beyang
+/cmd/frontend/internal/goroutine @keegancsmith
+/cmd/frontend/internal/inventory @beyang
+/cmd/frontend/internal/cli/middleware @beyang
+/cmd/frontend/internal/cli @beyang
+/cmd/frontend/internal/pkg/siteid @beyang
+/cmd/frontend/internal/pkg/suspiciousnames @beyang
+/cmd/frontend/internal/pkg/markdown @beyang
+/cmd/frontend/internal/pkg/handlerutil @beyang
+/cmd/frontend/internal/httpapi @beyang
+/cmd/frontend/types @beyang
+/cmd/frontend/hooks @beyang
+/cmd/frontend/internal/ @beyang
+/pkg/randstring/ @beyang
+/pkg/pubsub/ @beyang
+/pkg/repotrackutil/ @beyang
+/pkg/atomicvalue/ @beyang
+/pkg/testutil/ @beyang
+/pkg/debugserver/ @beyang
+/pkg/vfsutil/ @beyang
+/pkg/gituri/ @beyang
+/pkg/db/ @keegancsmith
+/pkg/processrestart @keegancsmith
+/pkg/honey @keegancsmith
+/pkg/ratelimit @beyang
+/pkg/registry @chrismwendt
+/pkg/slack @beyang
+/pkg/prefixsuffixsaver @beyang
+/pkg/gosrc @beyang
+/pkg/txemail @beyang
+/gulpfile.ts @beyang
+/client @felixfbecker
+/renovate.json @felixfbecker
+/.stylelintrc.json @felixfbecker
+/graphql-schema-linter.config.js @felixfbecker
+/.prettierignore @felixfbecker
+/.github @beyang
+/.gitmodules @beyang
+/.gitattributes @beyang
+/.yarnrc @felixfbecker
+/CHANGELOG.md @beyang
+/pkg/httputil @beyang
+/pkg/diskcache @beyang
+/pkg/sysreq @beyang
+/pkg/errcode @beyang
+/pkg/extsvc @beyang
+/pkg/routevar @beyang
+/pkg/env @beyang
+/pkg/pathmatch @beyang
+/pkg/version @beyang
+/pkg/httptestutil @beyang
+/pkg/mutablelimiter @beyang
+/pkg/buildkite @ggilmore
+/pkg/httpcli @beyang
+/packages @beyang
+/cmd/frontend @beyang
+/dev @beyang
+/.buildkite @beyang
+/.storybook @felixfbecker
+/CONTRIBUTING.md @beyang @nicksnyder
+/.dockerignore @beyang
+/.mailmap @beyang
+/tsconfig.json @felixfbecker
+/jest.config.base.js @felixfbecker
+/tslint.json @felixfbecker
+/enterprise/cmd @beyang
+/enterprise/pkg @beyang
+/enterprise @beyang
+/doc.go @beyang
+/.gitignore @beyang
+/prettier.config.js @felixfbecker
+/.editorconfig @felixfbecker
+/jest.config.js @felixfbecker
+/go.mod @beyang
+/cmd @beyang
+/pkg @beyang
+
+# Frontend
 /web/src/ @felixfbecker
+/shared @felixfbecker
+/web @felixfbecker @lguychard
+/ui @felixfbecker @lguychard
+/enterprise/ui @felixfbecker @lguychard
+/cmd/frontend/internal/app/jscontext @felixfbecker
+/packages/@sourcegraph @felixfbecker
```

```suggestion
/packages @sourcegraph/code-nav
```
Should include all packages